gds/src/main/scala/edu/gemini/aspen/gds/configuration/KeywordConfigurationFile.scala

```diff
@@ -0,0 +1,40 @@
+package edu.gemini.aspen.gds.configuration
+
+import edu.gemini.aspen.gds.configuration.KeywordConfigurationItem
+import util.parsing.input.Position
+
+/**
+ * This object provides utility methods to manipulate a configuration file
+ * Adapted from GDSConfigurationFile in previous (actor) version of GDS, with all
+ * file writing/updating removed.
+ */
+object KeywordConfigurationFile {
+  // TODO: This assumes no parsing errors and returns an empty list if there is. There was a
+  // gds-config-validator project that I guess was used for validation. I think we should do
+  // something to force validation, even if it is an exception. Successfully starting with no
```

Comment:
Yeah I think we should kill the GMP or at least the bundle if the config is bogus

Reply:
OK. I'll change it to log an error and exit if the config file is bad. I just wasn't sure if it was a design decision to carry on without the config file.